.github/workflows/maven-build.yml

```diff
@@ -72,3 +72,36 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
+
+  build-docs:
+    runs-on: ubuntu-latest
+    needs: build-and-test
+    container: node:16-alpine
+    steps:
+      - name: Install Dependencies
+        run: |
+          apk add tar 
+
+      - name: Cache nodejs deps
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: npm
+
+      - uses: actions/checkout@v4
+
+      - name: create .npmrc
+        run: | 
+          rm -rf ~/.npmrc
+          echo "@zepben:registry=${{ secrets.NEXUS_NPM_REPO }}" >> ~/.npmrc
+          echo "//mavenrepo.zepben.com/repository/zepben-npm/:_authToken=${{ secrets.CI_NPM_TOKEN }}" >> ~/.npmrc
+          echo "//npm.pkg.github.com/:_authToken=${{ secrets.CI_GITHUB_TOKEN }}" >> ~/.npmrc
+          echo "\n" >> ~/.npmrc
+
+      - name: Build docs
+        run: |
+          if [ -d docs ]; then
```

Comment:
this check could be done as part of build-and-test, but you could just save the result as an output, and then you should be able to apply the if condition to the build-docs job. That will save us running the container and a bunch of steps just to find out we don't have any docs. Can you make this change in each build?

Reply:
yup, can do